### ISSUE TITLE
AttributeError 'Bot' object has no attribute 'base_url' for bot.send_image

### ISSUE THREAD

**Issue by pseudonymcp** [davidchua/pymessenger#17]
Hi, I wanted to send an image as a file, not via url, but got an AttributeError 'Bot' object has no attribute 'base_url' for bot.send_image.

```
Traceback (most recent call last):
  File "paul.py", line 166, in webhook
    bot.send_image(sender_id, generated_image)
  File "/home/psuser/paul/paul/local/lib/python2.7/site-packages/pymessenger/bot.py", line 135, in send_image
    return requests.post(self.base_url, data=multipart_data, headers=multipart_header).json()
AttributeError: 'Bot' object has no attribute 'base_url'
```

**Comment by Cretezy** [comment]
Hey, looks like that function was never actually tested. I've been working on a minor rewrite that works with local attachments. Try using the [partial-rewrite](https://github.com/davidchua/pymessenger/tree/partial-rewrite) branch!

**Comment by Cretezy** [comment]
Fixed in #18
